### DISCUSSION TITLE
Timestamp skew in long audio

### DISCUSSION THREAD

**Discussion by tortoiseknightma** [openai/whisper#2086] [Q&A] [answered]
I'm trying to transcribe an audio from a movie, which is about 2.5h. I'm using `large-v3`

I've noticed timestamp skews in some parts of the scripts, which are usually accompanied by a series of duplicated words. 

As far as I know, chunking is automatically done by whisper with long audio. Still is there any connection between the problem and the length of my audio?

**Comment by glangford** [comment, marked as answer]
* https://github.com/openai/whisper/discussions/1811

**Comment by LaurinmyReha** [comment]
https://github.com/nyrahealth/CrisperWhisper this could help
